tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -352,13 +365,25 @@ void DebuggerProbesInstrumentationRequester::AddLineProbes(
         {
             const DebuggerLineProbeDefinition& current = lineProbes[i];
 
+            if (IsProbeIdExist(current.probeId))
+            {
+                Logger::Info("[AddLineProbes] Method Probe Id: ", current.probeId, " is already processed.");
+                continue;
+            }
+
             const shared::WSTRING& probeId = shared::WSTRING(current.probeId);
             const shared::WSTRING& probeFilePath = shared::WSTRING(current.probeFilePath);
             const auto& lineProbe = std::make_shared<LineProbeDefinition>(LineProbeDefinition(probeId, current.bytecodeOffset, current.lineNumber, current.mvid, current.methodId, probeFilePath));
             
             lineProbeDefinitions.push_back(lineProbe);
         }
 
+        if (lineProbeDefinitions.empty())
+        {
+            Logger::Info("[AddLineProbes] Early exiting, there are no new line probes to be added.");
```

Comment:
Should this be `DEBUG`?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.